acl/forgot_form.cgi

```diff
@@ -13,27 +13,25 @@ our (%in, %text);
 &can_edit_user($in{'user'}) || &error($text{'edit_euser'});
 my $u = &get_user($in{'user'});
 $u || &error($text{'edit_egone'});
-
+my $is_admin = $u->{'name'} =~ /^(?:root|admin|sysadm)$/;
```

Comment:
Only the `root` user can be sudo'd to though. Also, hard-coding usernames in general is a bad idea.

Reply:
Right, but sudo isn't applicable to any users other than `root` ?

Reply:
Yes, but a sudo-capable user won't be able to login as "admin"

Reply:
Okay, I have reverted to using the previous test, and then later, as you mentioned, we will fix it properly, because neither tests are accurate enough!

Reply:
In this case it is definitively "root", per the code at https://github.com/webmin/webmin/blob/master/miniserv.pl#L3893

Reply:
Okay… but why is this code even relevant? We’re using a Webmin user to generate a password reset link for another user. Any Webmin non-safe user with access to ACL and Users and Groups module should be able to do that, right?